### PR TITLE
Signing with request with SHA256 signature algorithm and changing version to Cybersource-sdk-java 6.1.0

### PR DESCRIPTION
**Pull request by mahendya**
Hi ,

Firstly i am bringing Future branch to same level as Master branch as both the branch are not sync.

My changes:-
•	Modified soapWrapAndSign method in ApacheSignatureWrapper.java to support SHA256    Signature  and Digest Algorithm 
•	Added client lib version in utility.java.(From 5.1.1 to 6.1.0 )
•	And modified targetAPIVersion to 1.123(latest from the NTA server)( from 1.119 to 1.123).This is in cybs.properties file.
•	Modify pom.xml in nvp and xml sample projects for cybs_sdk_java version(6.1.0) and in sample      project ,we are getting JCE authentication error (Due to bcprov-jdk16-1.46.jar). bcprov-jdk16-1.46.jar is a signed jar, and when we unpack it and include in our own jar file, then Java fails to validate the signature.  It's better to include it separately in the CLASSPATH.Hence using maven dependency pluging to copy dependencies and adding them to CLASSPATH
•	In ReadMe file, modified version in maven and Gradle dependency co-ordinates.

Regards,
Mahendra Yadav

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -9,7 +9,7 @@
     </parent>
     <groupId>com.cybersource</groupId>
     <artifactId>cybersource-sdk-java</artifactId>
-    <version>6.0.2-SNAPSHOT</version>
+    <version>6.1.0</version>
```
Comment (dhimandeepak) [inline pom.xml]: Keep the version as snapshot, we will use maven release plugin to build next version

File: src/test/resources/test_cybs.properties [hunk 2]
```diff
@@ -12,7 +12,7 @@ timeout=1000
 # logging should normally be disabled in production as it would slow down the
 # processing.  Enable it only when troubleshooting an issue.
 enableLog=true
-logDirectory=<-- Directory where cybs.log file needs to be created -->
+logDirectory=/Users/anetdeveloper/logs
 logMaximumSize=10
```
Comment (dhimandeepak) [inline src/test/resources/test_cybs.properties]: Remove data from properties

File: pom.xml (hunk 1 above)
Comment (mahendya) [inline pom.xml]: Thank you for reviewing.Version changed to 6.1.0-SNAPSHOT.

File: src/test/resources/test_cybs.properties (hunk 2 above)
Comment (mahendya) [inline src/test/resources/test_cybs.properties]: As part of our changes,we have not touched this file.
Anyway removed that data from the properties.

**Pull request merged**